<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqLookingGlassDockPanel</class>
 <widget class="QWidget" name="pqLookingGlassDockPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Looking Glass</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="RenderOnLookingGlassButton">
     <property name="text">
      <string>Render Active View on Looking Glass</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="TargetDeviceLayout">
     <item>
      <widget class="QLabel" name="TargetDeviceLabel">
       <property name="text">
        <string>Target Device:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="TargetDeviceComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="ResetToCenterOfRotationButton">
     <property name="text">
      <string>Reset to Center of Rotation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="PushFocalPlaneBackButton">
     <property name="text">
      <string>Push Focal Plane Back</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="PullFocalPlaneForwardButton">
     <property name="text">
      <string>Pull Focal Plane Forward</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="SaveQuilt">
     <property name="text">
      <string>Save Quilt</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="RecordQuilt">
     <property name="text">
      <string>Record Quilt</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="FocalDistanceLayout">
     <item>
      <widget class="QLabel" name="FocalDistanceLabel">
       <property name="text">
        <string>Focal Distance:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="pqDoubleRangeWidget" name="FocalDistance" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>137</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqDoubleRangeWidget</class>
   <extends>QWidget</extends>
   <header>pqDoubleRangeWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
